web/package-lock.json

```diff
@@ -2330,8 +2330,7 @@
     "deep-equal": {
```

Comment:
this comment isn't related to this specific line but Github flagged a potential security vulnerability in this file. Could you take a look at the below and if there's a simple fix, apply it?

![image](https://user-images.githubusercontent.com/15616604/37638653-436dae4c-2bcb-11e8-853e-710e55549c68.png)

Reply:
I've updated these packages to the latest version so the warnings should go away.